Mathlib/Analysis/InnerProductSpace/MulOpposite.lean

```diff
@@ -26,38 +26,51 @@ instance [Inner 𝕜 H] : Inner 𝕜 Hᵐᵒᵖ where inner x y := inner 𝕜 x.
 
 @[simp] theorem inner_op [Inner 𝕜 H] (x y : H) : inner 𝕜 (op x) (op y) = inner 𝕜 x y := rfl
 
-variable [RCLike 𝕜] [NormedAddCommGroup H] [InnerProductSpace 𝕜 H]
+variable [SeminormedAddCommGroup H]
+
+section InnerProductSpace
+variable [RCLike 𝕜] [InnerProductSpace 𝕜 H]
```

Comment:
This results in a slightly weird variable order; I'd suggest
```suggestion
section InnerProductSpace
variable [RCLike 𝕜] [SeminormedAddCommGroup H] [InnerProductSpace 𝕜 H]
```
and just reintroduce the variable below after `Semiring R`.